### PR TITLE
Added SentimentAnalyzer project for up for grabs

### PR DESCRIPTION
**Pull request by arafattehsin**

### CODE REVIEW

File: SentimentAnalyzer.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: SentimentAnalyzer
+desc: Semantic Analyzer is an on-device (offline) open-source library to find out what customers think of your brand or topic by analyzing raw text for clues about positive or negative sentiment. This library returns a sentiment probability and score along with the prediction flags (true for Positive, false for Negative). Powered by Microsoft ML.NET
```
Comment (ritwik12) [inline SentimentAnalyzer.yml]: @arafattehsin Want to make it a bit short?

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@arafattehsin Nice. I once developed a Sentiment Analysis model a couple of years back. Did that using CNN for Sentence This stuff is tough and not enough clean data to train the model.

**Comment by arafattehsin** [comment]
@ritwik12 Thanks! I just edited the description.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@arafattehsin We reverted this as this was not in _data/projects folder. Please open a new PR.

**Pull request merged**